web/src/components/navbar/NavBarMobile.tsx

```diff
@@ -34,15 +35,33 @@ export const NavBarMobile = (props: Props): ReactElement => {
   const isAuthenticated = useMemo(() => {
     return state.isAuthenticated === "TRUE";
   }, [state.isAuthenticated]);
-  const userName = getUserNameOrEmail(userData);
-  const textColor = isAuthenticated ? "primary" : "base";
-  const accountIcon = isAuthenticated ? "account_circle" : "help";
-  const accountString = isAuthenticated ? userName : Config.navigationDefaults.navBarGuestText;
+
+  const currentlyOnboarding = (): boolean => {
+    if (!router) {
+      return false;
+    }
+    return router.pathname === ROUTES.onboarding;
+  };
+
+  const getMenuConfirguration = (): menuConfiguration => {
+    if (props.isLanding) {
+      return "login-getstarted";
+    }
+    if (currentlyOnboarding()) {
+      return "login";
+    } else {
+      if (isAuthenticated) {
```

Comment:
this could be another else-if right, doesn't need to be nested

Reply:
yes